### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mikethreels**
Changelog

- Added top navbar
- Added main paragraph section
- Added pictures
- Added video section
- Added aside section
- Added footer section

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+                </a>
+                <a href="#">
+                    <div class="topic-description">
+                        <img src="images/analog-clock.jpg" alt="Analog clock">
+                        <div class="mini-title-container">
+                            <span class="mini-title washed-out flex-end">Paul Kooiker for The New York Times</span>
+                        </div>
+                        <h3 class="main-title customfont">How Analog Clocks Can Give Us More by Giving Us Less</h3>
+                        <span class="date-posted">May 5</span>
+                    </div>
```
Comment (addod19) [inline index.html]: We could improve semantics by using ```<article>``` instead of ```< div class="topic-description">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi, @mikethreels and @mimipeshy, great job on this project :+1: congratulations. Let us improve our code and make it better. Kindly find comments below: :smile: 

- The text should have a link, currently, it does not
last
- Kindly add a detailed description to the README. You can check pathwright for the description, for example 'use float, flex, and grid to position elements.
- The live link is currently not working properly, kindly add more social handles to the author section
- Let us apply the DRY rule to our CSS file, so we avoid repeating code. This [article](https://speckyboy.com/good-bad-css-practices/)

KIndly make these changes and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Larbi](https://speckyboy.com/good-bad-css-practices/)

**Comment by mikethreels** [comment]
Changelog
- updated hamburger button, search icon and the text in the navbar.
- updated more detailed README file.
- fixed the live link.
- cleaned up the CSS.
- replaced the `div class="topic-description"` to `article`

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

#### Great job on implementing the changes!

#### Your project looks great!

### You can go ahead and merge this pull request!

### Happy coding! :computer:

## Lucas Mazo

**Pull request merged**